src/main/java/com/redhat/cloud/notifications/GwResource.java

```diff
@@ -137,4 +143,12 @@ public static String serializeAction(Action action) throws IOException {
                 .build();
         return Message.of(payload).addMetadata(metadata);
     }
+
+    boolean isBundleApplicationEventTypeTripleValid(String bundle, String application, String eventType) {
+        try {
+            return restValidationClient.isBundleApplicationEventTypeTripleValid(bundle, application, eventType).getStatus() == 200;
+        } catch(ResteasyWebApplicationException e) {
```

Comment:
What happens if another kind of exception is thrown? What kind of response will the notifications-gw client get?

Reply:
I think this is generally ok, but should be a `WebApplicationException` and not the RE specific one. 
All the other exceptions are then subclasses.